packages/devtools_app/lib/src/memory/flutter/memory_heatmap.dart

```diff
@@ -0,0 +1,497 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:math' as math;
+import 'dart:ui';
+
+import 'package:flutter/gestures.dart';
+import 'package:flutter/material.dart' hide TextStyle;
+import 'package:flutter/rendering.dart' hide TextStyle;
+import 'package:flutter/widgets.dart' hide TextStyle;
+import 'package:intl/intl.dart';
+
+import 'memory_graph_model.dart';
+
+class FlameChart extends StatelessWidget {
+  const FlameChart(
+    this.sizes, {
+    // Flame chart has a blueish color.
+    this.lightColor = const Color(0xFFBBDEFB),
```

Comment:
Add these colors as constants to
https://github.com/flutter/devtools/blob/3ff20fc3d75fdba37f5340c0a3f96858f52e9ab2/packages/devtools_app/lib/src/ui/colors.dart
that way we can make sure we use identical colors for all cases we are doing the same thing. For example. can we reuse some of the existing flame chart colors used by the timeline.

Reply:
Added:

/// Memory heat map blueish color from Blue 100 to Blue 900.
Color memoryHeatMapLightColor = const Color(0xFFBBDEFB);  // Material design BLUE 100
Color memoryHeatMapDarkColor = const Color(0xFF0D47A1);   // Material design BLUE 900

I agree should look at either mimicing or replacing to match flame chart colors and navigation.  For now I need a larger range of colors to see differences in the heat map.